tests/pypi/extension/extension_tests.bzl

```diff
@@ -351,16 +355,19 @@ def _test_download_only_multiple(env):
 --implementation=cp
 --abi=cp315
 
-simple==0.0.1 --hash=sha256:deadbeef
-extra==0.0.1 --hash=sha256:deadb00f
+simple==0.0.1 \
+    --hash=sha256:deadbeef
+extra==0.0.1 \
+    --hash=sha256:deadb00f
```

Comment:
Note, this formatting of putting the hashes on a new line is a more realistic representation how the actual lock files look like.